Documentation/ceph-cluster-crd.md

```diff
@@ -491,7 +491,8 @@ spec:
     nodes:
     - name: "172.17.4.201"
       devices:             # specific devices to use for storage can be specified for each node
-      - name: "sdb"
+      - name: "sdb" # Whole storage device
+      - name: "sdb2" # One specific partition. Should not have a file system on it.
```

Comment:
How about `sdc1` so the partition belongs to a different disk than the one on the previous line that is supposed to be consuming the whole device?